website/content/installation/_index.md

```diff
@@ -75,7 +75,7 @@ If you want to deploy MetalLB using the [experimental FRR-K8s mode]({{% relref "
 kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.14.3/config/manifests/metallb-frr-k8s.yaml
 ```
 
-Please do note that these manifests deploy MetalLB from the main development branch. We highly encourage cloud operators to deploy a stable released version of MetalLB on production environments!
+These manifests deploy MetalLB from a release branch. Indeed, we highly encourage cloud operators to deploy a stable released version of MetalLB (instead of manifests from the main branch) on production environments!
```

Comment:
Sorry I finally got here. I think the right approach is just to remove this part as a whole, and it should be reomved from the main branch as it is the master where we generate all the versioned release notes using this script https://github.com/metallb/metallb/blob/25f1bbcfa9f86675e492ccbc29459ba55fb69609/tasks.py#L707

Reply:
Hi @fedepaol, so do you mean to remove the line 78 entirely ?

Reply:
Yep, as it sounds like a legacy bit from where we were not replacing main with the right tag after a release.